pyretailscience/plots/period_on_period.py

```diff
@@ -0,0 +1,119 @@
+"""Period on period module.
+
+This module provides functionality for plotting multiple overlapping time periods
+from the same time series on a single line chart using matplotlib.
+
+The `plot` function is useful for visual comparisons of temporal trends
+across different time windows, with each time window plotted as a separate line
+but aligned to a common starting point.
+
+Example use case: Comparing sales data across multiple promotional weeks or seasonal periods.
+"""
+
+import matplotlib.pyplot as plt
+import pandas as pd
+from matplotlib.axes import Axes
+
+import pyretailscience.style.graph_utils as gu
+from pyretailscience.plots.line import plot
+
+LINE_STYLES = [
+    "-",  # solid
+    "--",  # dashed
+    ":",  # dotted
+    "-.",  # dashdot
+    (0, (5, 10)),  # long dash with offset
+    (0, (6, 6)),  # loosely dashed
+    (0, (3, 5, 1, 5)),  # loosely dashdotted
+    (0, (3, 5, 1, 5, 1, 5)),  # loosely dashdotdotted
+]
+
+
+def overlapping_periods(
+    df: pd.DataFrame,
+    x_col: str,
+    y_col: str,
```

Comment:
We typically use ˋvalue_colˋ instead of ˋy_colˋ. Can you please change it, but you make me wonder whether we should be using ˋ y_colˋ everywhere. Any thoughts?
```suggestion
    value_col: str,
```

Reply:
I think we should use `value_col` instead of `y_col` for consistency, since other functions like `plot()` already use `value_col`. It's more general-purpose and keeps the API unified, especially if the function is reused for different types of plots.